packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -900,6 +900,15 @@ - (void)invalidate {
   
 }
 
+RCT_EXPORT_METHOD(setTenantID:
+  (FIRApp *) firebaseApp
+    :(NSString *) tenantID
+) {
+    if(tenantID){
+        [FIRAuth authWithApp:firebaseApp].tenantID = tenantID;
+    }
```

Comment:
If you are checking for not-null and that it is a valid string in JS, I think you can skip the if(tenantId) here?

```suggestion
    [FIRAuth authWithApp:firebaseApp].tenantID = tenantID;
```